### ISSUE TITLE
Adding non-English resources to a themed list?

### ISSUE THREAD

**Issue by ramnes** [sindresorhus/awesome#1282]
Please see this issue on Awesome MongoDB: https://github.com/ramnes/awesome-mongodb/issues/43 

By reading #153 and #1176, I seem to understand that all the non-English resources are supposed to go in separated lists, rather than being regrouped under the main theme list, even if they're not translations but really add to the list.

Can't we do better and be more inclusive to non-English speakers? For example, by adding a country flag in front of all resources, indicating the language of the resource?

**Comment by sindresorhus** [comment]
I don't think it makes sense to mix languages in the main readme. That just creates a messy experience for everyone.

Instead, make the main readme.md have only English content, then at the top of that readme, add links to other languages which can be in their own Markdown files.

For example:

```
readme.md
readme-jp.md
readme-no.md
```

When we've decided on a good format, we should document it here.

**Comment by sindresorhus** [comment]
// @qazbnm456

**Comment by zhaoyi0113** [comment]
The idea on below format makes sense for translating into multiple languages. But what we want to is to bring articles written natively by other languages instead of translation. 

```
readme.md
readme-jp.md
readme-no.md
```

**Comment by sindresorhus** [comment]
@zhaoyi0113 That's what I'm talking about too.

**Comment by ramnes** [comment]
I feel that the different README files would be interesting if there would be a lot of resources in the different languages. But most of the time that's just not the case. There's maybe 95% of the resources available in English, since English is still prominent in computer sciences, and 5% of very specific stuff not already covered by the 95% and written in some other other language.

In the age of evolved translators like DeepL, I don't see any reason to not add an article written in another language if it really adds to the list. Non-English articles could be enforced to have at least one translation, with a guideline on how to format translations.

For example, something in those lines would make sense to me:
- Great resource - Description
- 巨大的资源 (:cn: Some other great resource) - Description in English (translations: :gb:)

I think such a format would also be very useful for local lists, non related to computer science, where the main language might be something else than English and with resources in different languages.

What do you think @sindresorhus?

**Comment by sindresorhus** [comment]
> In the age of evolved translators like DeepL, I don't see any reason to not add an article written in another language if it really adds to the list.

I have yet to see a technical paper in Chinese being readable after running it through Google Translate, but I don't see why not if it's still good after being translated.

> 巨大的资源 (🇨🇳 Some other great resource) - Description in English (translations: 🇬🇧)

Since the list is in English, the main link should point to the translated version. The original can be linked to in parens, like *([Source]())*.

**Comment by ramnes** [comment]
> Since the list is in English, the main link should point to the translated version. The original can be linked to in parens, like (Source).

Yep, and it has the advantage of forcing a proper translation in the list's main language as the main link. I definitely support that solution.

**Comment by jirayutza1** [comment]
> readme.md
> readme-jp.md
> readme-no.md

Duplicate of #1664

**Comment by touxiongjihua** [comment]
1

**Comment by Tester1234537** [comment]
What is the problen con the spanish

**Comment by Angel-Mathew007** [comment]
Hi everyone,

I find this discussion about including non-English resources very interesting and important for making the lists more inclusive. I’d love to help contribute to improving how non-English resources are added or organized.

Could someone guide me on how to get started with this? Is there an existing guideline or format I should follow when adding such resources?

Thanks!

**Comment by 5npyrh9kvj-sms** [comment]
- [ ]

**Comment by Ke639** [comment]
Line
